packages/mobile/android/app/build.gradle

```diff
@@ -144,6 +144,13 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
     flavorDimensions "default"
 
+    packagingOptions {
```

Comment:
wild, whats the tldr here?

Reply:
Oh this was just to fix [this issue](https://github.com/arthenica/ffmpeg-kit/wiki/Tips#2-depending-another-android-library-containing-libc_sharedso) where two of our packages included `libc++_shared.so`